src/test/java/com/wikia/webdriver/elements/oasis/components/notifications/Notification.java

```diff
@@ -0,0 +1,49 @@
+package com.wikia.webdriver.elements.oasis.components.notifications;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.SpecialRestorePageObject;
+import jdk.nashorn.internal.objects.NativeRegExp;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.openqa.selenium.support.FindBy'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)